server/heroes/heroCtrl.js

```diff
@@ -29,5 +28,47 @@ module.exports = {
     //TODO: interface with db, always sends true
     res.status(201).send(true);
 
+  }, 
+  acceptRequest: function(req, res, next) {
+    //get transaction id from request
+    var transactionId = req.body.transactionId;
+
+    //TODO: update status of transaction in db
+    //from unfulfilled to in progress
+
+    res.status(201).send('You have accepted a request');
+  }, 
+  getOpenRequests: function(req, res, next) {
+    //get location from request
+    var location = req.query.location;
+
+    //TODO: find all transactions with location = ^
+
+    //dummy data (needs to be gotten from the db)
+    var orders = [{
+        time: "2015-10-02T05:20:58.409Z",
```

Comment:
sometimes strings are `""` and sometimes `''`. Javascript strings should be single quotes